tests/c_api_tests/test_cpp_api_model.cpp

```diff
@@ -134,14 +159,22 @@ TEST_CASE("C++ API Model") {
     load_updates_buffer.set_value(PGM_def_update_sym_load_id, load_updates_id.data(), -1);
     load_updates_buffer.set_value(PGM_def_update_sym_load_q_specified, load_updates_q_specified.data(), 0, -1);
     load_updates_buffer.set_value(PGM_def_update_sym_load_q_specified, load_updates_q_specified.data(), 1, -1);
-
     // dataset
-    DatasetConst single_update_dataset{"update", 0, 1};
+    DatasetConst const single_update_dataset{"update", 0, 1};
     single_update_dataset.add_buffer("source", 1, 1, nullptr, source_update_buffer);
     single_update_dataset.add_buffer("sym_load", 1, 1, nullptr, load_updates_buffer.get());
-    DatasetConst batch_update_dataset{"update", 1, 2};
+    // update containing columnar data still fail
```

Comment:
I hope this line is to be removed?

Reply:
Indeed.

Reply:
Addressed in the latest commit. Comment resolved.